src/main/java/mate/academy/springboot/datajpa/repository/ProductRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.datajpa.repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    List<Product> findProductByPriceBetween(BigDecimal from, BigDecimal to);
+
+    List<Product> findProductsByCategoryNameIn(List<String> categories);

```

Comment:
Let's be more attentive to mathod naming, and swap list to set
```suggestion
    List<Product> findAllProductsByPriceBetween(BigDecimal from, BigDecimal to);

    List<Product> findAllProductsByCategoryNameIn(Set<String> categories);
```